vignettes/getting-started.Rmd

```diff
@@ -16,15 +16,27 @@ knitr::opts_chunk$set(
 
 # gimap tutorial
 
+TODO: Describe what this genetic interaction analysis actually is and why someone would want to do it. 
+
 ```{r}
 library(magrittr)
 library(gimap)
 ```
 
+## Data requirements 
+
+TODO: What kind of data would someone need to run this? How much flexibility is there in what the experimental set up might look like? 
```

Comment:
@sobrien29 this might be a good question to address with Alice and the broader team; e.g. how many timepoints are required for minimum functionality, will we require the pgPEN library, etc

Reply:
I generally agree with the idea let's develop this for the Berger lab first and then we can try to expand this package's flexibility as we move along. But totally, this is probably a bigger and ongoing discussion amongst the team!

Reply:
@sobrien29 How would you describe this to a user in this context? What is a T0? That means just the first time point at 0 days? Do you have a paper we can link to that give people the basics of CRISPR screens?